Dependencies.props

```diff
@@ -27,4 +27,14 @@
     <Xunit_Assert_Version>2.4.1</Xunit_Assert_Version>
     <Xunit_Core_Version>2.4.1</Xunit_Core_Version>
   </PropertyGroup>
+  <PropertyGroup Label="Packages for Omex Extensions. AutoUpdate">
+    <Microsoft_Extensions_Logging_Abstractions_Version>3.1.0</Microsoft_Extensions_Logging_Abstractions_Version>
```

Comment:
>Microsoft_Extensions_Logging_Abstractions_Version [](start = 5, length = 49)

Would we always want to keep some of these packages on the same version? For instance, will the first four always have the same version? If so, a single variable would be better to ensure this is maintained

Reply:
Yes, it makes sense, my hope in splitting them is that we will start using the same system for automatic package update as in internal repositories and this system uses the variable name to find the latest version so they have to be separated.